### ISSUE TITLE
solidarite-numerique.fr

### ISSUE THREAD

**Issue by JulienMoraliBSF** [openzim/zim-requests#447]
*Please use the following format for a ZIM creation request (and delete unnecessary information)*

- Website URL: https://www.solidarite-numerique.fr/
- License: Public domain ?
- Desired ZIM Title: Solidarité numérique : La plateforme d'aide pour mes besoins numériques du quotidien
- Desired ZIM Description: Plateforme d'aide pour les besoins numériques du quotidien (tutoriels)
- Desired ZIM Icon –png (URL or attach one): https://www.solidarite-numerique.fr/wp-content/themes/snum-v2/images/logoSN.svg
- Language (ISO 639-3): **fr**
- Desired Main Page (homepage, if different from website URL): https://www.solidarite-numerique.fr/
- Is this a MediaWiki?:

**Comment by JulienMoraliBSF** [comment]
Hello @Popolechien @rgaudin 
Any update for this ZIM request ?
Our content team deployment is scheduled early in October, can it be avalaible then ?

**Comment by Popolechien** [comment]
@JulienMoraliBSF I created https://farm.openzim.org/recipes/solidarite-numerique - the current queue is fairly long but if it works we should be good (big "if" though).

**Comment by Popolechien** [comment]
Task is listed as succeeded, but from what I can see it only picked up the front page (and then some, as file size is 223MB).

However clicking on any link returns "This content is blocked". 
![Capture d’écran 2022-09-15 à 09 14 01](https://user-images.githubusercontent.com/21675885/190339424-48da72e0-5bf9-40ba-a634-9ce35765a2ca.png)
@rgaudin That's a new one for me. Any idea why that is?

**Comment by RavanJAltaie** [comment]
@Popolechien if you click on ([lire la suite](https://dev.library.kiwix.org/content/solidarite-numerique_fr_2023-03/A/mp_/https://www.solidarite-numerique.fr/tutoriels/supprimer-les-cookies-sur-les-navigateurs-firefox-chrome-edge/?thematique=internet) the sub-page opens normally.
I checked all of them.
@rgaudin please check if this is solvable.

**Comment by Popolechien** [comment]
Woah woah, that's problematic.
Testing on https://dev.library.kiwix.org/viewer#solidarite-numerique_fr_2023-03/A/www.solidarite-numerique.fr/

The following items are available and clickable:
![Capture d’écran 2023-06-27 à 09 49 25](https://github.com/openzim/zim-requests/assets/21675885/19df9bc2-dd72-4764-9432-f5109fc9d727)

- Clicking on the image leads to the error message above
- Clicking on `Lire la suite` leads to the expected page.

Bottom line: same link, different behaviour.

**Comment by rgaudin** [comment]
That's a kiwix-serve+Chrome issue. Please test with Firefox until this is figured out.

**Comment by rgaudin** [comment]
Thanks for the report ; https://github.com/openzim/zimit/issues/196

**Comment by benoit74** [comment]
Last task : https://farm.openzim.org/pipeline/ef3ebf25-0752-4f8a-83a1-a33ed72ce191

Last task (and the one before) failed due to an incorrect `wait` value: `load,network0` is not allowed anymore, it should probably be replaced by `load,networkidle` (but do we really need this setting ? does anyone (@rgaudin ?) remembers why it has been put here ?

I'll take over the setup of this recipe for now to check how it runs with `zimit2` since this is clearly one case which would benefit from the update: @RavanJAltaie @Popolechien please add this to the list of zimit2 tasks you should not touch anymore for now.

**Comment by benoit74** [comment]
I've removed the custom icon since the website favicon is really fine, updated to zimit2, renamed the ZIM and recipe to match our convention, removed the `wait` setting and launched the recipe

New recipe location: https://farm.openzim.org/recipes/www.solidarite-numerique.fr_fr_all

**Comment by benoit74** [comment]
A new zimit2 ZIM has been built and is available at https://dev.library.kiwix.org/viewer#www.solidarite-numerique.fr_fr_all_2024-05 or searchable with https://dev.library.kiwix.org/?q=slid#lang=fra&q=solid 

Recipe is running again because I had to hide few more things with CSS, but the result seems pretty solid (it even reproduce with high fidelity graphical bugs present on original website !).

**Comment by Jaifroid** [comment]
Congrats @benoit74, this ZIM is not showing any character encoding issues in the PWA on a quick test! No problems clicking any of the tiles / links that I tried.

The only thing is video: on the "Lire les Cookies" page there is a video that shows as unavailable in both Firefox and Chrome. I don't think this is the same issue as we saw with 100-rabits.

![image](https://github.com/openzim/zim-requests/assets/4304337/525b215a-d011-42f2-b6e4-d79461c7741e)

**Comment by benoit74** [comment]
Yep, this video is not available on source website, so it is indeed a different issue 😅

https://www.solidarite-numerique.fr/tutoriels/comprendre-les-cookies/

Thank you!

**Comment by Jaifroid** [comment]
> Yep, this video is not available on source website, so it is indeed a different issue 😅

Sounds like we need a workaorund for such cases. All we have to do is incorporate offline [Sora](https://openai.com/index/sora/) to generate the missing video... 🤣

**Comment by benoit74** [comment]
ZIM has been published to production with Zimit2, it is now working perfectly as far as I can tell 🎉

- Kiwix-serve: https://library.kiwix.org/viewer#www.solidarite-numerique.fr_fr_all_2024-06 
- Library search: https://library.kiwix.org/#lang=&q=solidarit%C3%A9 
- Direct download: https://download.kiwix.org/zim/zimit/www.solidarite-numerique.fr_fr_all_2024-06.zim

**Comment by benoit74** [comment]
(aside from bug present on source website of course, but at least they are reproduced with high fidelity ^^)

**Comment by Jaifroid** [comment]
Quick query, are we starting to publish Zimit2 ZIMs publicly? Or is this a special case?

It seems to me that it would be good to have a launch that is advertised, at least on social media channels, to let people know that x, y, z ZIMs are now available with shiny new Zimit2 and that they work on most readers where Zimit1 didn't work before.

**Comment by benoit74** [comment]
File is ready in prod since June.

https://library.kiwix.org/#lang=fra&q=solidarit%C3%A9

Recipe is set to biannually update since (at least ATM) there is no much updates ongoing.
